tests/assets/add-known-host-missmatch.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
```

Comment:
I just realized this one can be done within the original `add-known-host.sh`
I will do that after hearing comments back.